### PR TITLE
Add release notes and bump version for 0.17.0

### PR DESCRIPTION
**Pull request by jadchaar**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪 Added **tests** for changed code.
- [x] 🛠️ All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [x] 🧹 All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚 Updated **documentation** for changed code.
- [x] ⏩ Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

## Description of Changes

Changes summarized here: https://github.com/arrow-py/arrow/compare/0.16.0...master

### CODE REVIEW

**Review by krisfremen — APPROVED** [review]
Nice. :+1: 

One step closer to 1.0.0 release.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+    ...     print(r)
+    ...
+    (<Arrow [2018-03-10T23:00:00-08:00]>, <Arrow [2018-03-11T07:00:00+00:00]>)
+    (<Arrow [2018-03-11T00:00:00-08:00]>, <Arrow [2018-03-11T08:00:00+00:00]>)
+    (<Arrow [2018-03-11T01:00:00-08:00]>, <Arrow [2018-03-11T09:00:00+00:00]>)
+    (<Arrow [2018-03-11T03:00:00-07:00]>, <Arrow [2018-03-11T10:00:00+00:00]>)
+    (<Arrow [2018-03-11T04:00:00-07:00]>, <Arrow [2018-03-11T11:00:00+00:00]>)
+
+
+- [NEW] Implemented `PEP 495 <https://www.python.org/dev/peps/pep-0495/>`_ to handle ambiguous datetimes. This is achieved by the addition of the ``fold`` attribute for Arrow objects. For example:
```
Comment (systemcatch) [inline CHANGELOG.rst:29]: I think this has been duplicated from the last release notes?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+- [NEW] Added ``humanize`` week granularity translation for Tagalog.
+- [CHANGE] Calls to the ``timestamp`` property now emit a ``DeprecationWarning``. In a future release, ``timestamp`` will be changed to a function to align with Python's datetime module. If you would like to continue using the property, please change your code to use the ``int_timestamp`` or ``float_timestamp`` properties instead.
```
Comment (systemcatch) [inline CHANGELOG.rst:31]: Minor nitpick, method rather than function.

**Review by systemcatch — CHANGES_REQUESTED** [review]
Few comments from me, maybe we should mention the internal reordering changes as well.

File: CHANGELOG.rst (hunk 1 above)
Comment (jadchaar) [reply CHANGELOG.rst:29]: Oh yeah sorry, fixed!

**Review by systemcatch — APPROVED** [review]

**Comment by systemcatch** [comment]
Hey @jadchaar do we want to merge this and cut the release?

**Pull request merged**